src/main/java/com/onarandombox/MultiversePortals/commands/ListCommand.java

```diff
@@ -19,10 +20,12 @@
 
 public class ListCommand extends PortalCommand {
 
+    int itemsPerPage = 18;
+
     public ListCommand(MultiversePortals plugin) {
         super(plugin);
         this.setName("Portal Listing");
-        this.setCommandUsage("/mvp list " + ChatColor.GOLD + "[FILTER] [WORLD]");
+        this.setCommandUsage("/mvp list " + ChatColor.GOLD + "[FILTER/PAGE] [WORLD]");
```

Comment:
Ideally, it better to be `[filter] [page]`. the FILTER and WORLD can merge since it's both some form of filtering. 

So you can do it such that:
* if it's 1 args, try to convert it to int, else assume its filter.
* if it's 2 args, assume it's filter string followed by page.
* if the filter string is a world name, show portals in that world, else filter by portal name.